js/main.js

```diff
@@ -289,6 +290,83 @@ Site.Drawers.About = {
       $('#about-drawer-' + target).addClass('active');
     });
   }
-}
+};
+
+Site.Media = {
+  init: function() {
+    var _this = this;
+
+    _this.bind();
+  },
+
+  bind: function() {
+    var _this = this;
+
+    if ($('.page-sound-and-vision').length) {
```

Comment:
yh. Possibly there will be other media playing pages so figured to init for each page in time